src/kdalgorithms.h

```diff
@@ -127,6 +127,21 @@ Container sorted(Container container, Compare &&compare = {})
     return container;
 }
 
+template <typename Container, typename Member>
+void sort_by(Container &container, Member &&member)
+{
+    sort(container, [member = std::forward<Member>(member)](const auto &x, const auto &y) {
```

Comment:
Not much point in forwarding a pointer. It can be by-value. Other than that, patch is ok.